lib/RedisVidIndexGenerator.cpp

```diff
@@ -25,6 +27,16 @@ uint64_t RedisVidIndexGenerator::increment()
     return m_dbConnector->incr(m_vidCounterName); // "VIDCOUNTER"
 }
 
+uint64_t RedisVidIndexGenerator::incrementBy(uint64_t count)
```

Comment:
also fix convention, each param in new line with In prefx

Reply:
Done